src/reducers/createUserReducer.js

```diff
@@ -0,0 +1,18 @@
+import { RECV_DATA,RECV_ERROR,ALERT_SUCCESS,ALERT_FAIL } from '../actions/ActionTypes';
+
+export default function createUser(state=[],action){
```

Comment:
Why using state = []. I can not see any use of it. Are you clear about this use case of state here?